### PR TITLE
Serverless 1.10.0 release notes

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/serverless-rn-1-10-0.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* The annotation `knative-eventing-injection=enabled` that was previously used to annotate namespaces for broker creation is now deprecated. The new annotation is `eventing.knative.dev/injection=enable`. For more information, see the documentation on _Creating a broker as a cluster administrator using namespace annotation_.
+* Multi-container support is now available on Knative as a Technology Preview feature. You can enable multi-container support in the `config-features` config map.
+// For more information see...... need to add docs on this / link to upstream docs?
+
+[id="fixed-issues-1-10-0_{context}"]
+== Fixed issues
+* In previous releases, Knative Serving had a fixed, minimum CPU request of 25m for `queue-proxy`. If your cluster had any value set that conflicted with this, for example, if you had set a minimum CPU request for `defaultRequest` of more than `25m`, the Knative Service failed to deploy. This issue is fixed in 1.10.0.
+
+[id="known-issues-1-10-0_{context}"]
+== Known issues
```
Comment (mgencur) [inline modules/serverless-rn-1-10-0.adoc:19]: IMO, this issue should be documented as known: https://issues.redhat.com/browse/SRVKS-641

**Review by mgencur — COMMENTED** [review]
Should we also add the list of versions? Knative Serving, Eventing etc.
Aside from that, mentioned one issue for Known issues.

File: modules/serverless-deleting-broker-admin.adoc [hunk 2]
```diff
@@ -9,12 +9,12 @@ Brokers created due to annotation will not be removed if you remove the annotati
...
 .Procedure
 
-. Remove the `knative-eventing-injection=enabled` annotation from the namespace:
+. Remove the `eventing.knative.dev/injection=enable` annotation from the namespace:
 +
 
 [source,terminal]
 ----
-$ oc label namespace <NAMESPACE> knative-eventing-injection-
+$ oc label namespace <NAMESPACE> eventing.knative.dev/injection=enable
```
Comment (cardil) [inline modules/serverless-deleting-broker-admin.adoc:17]: This is invalid! It should be:

```
$ oc label namespace <NAMESPACE> eventing.knative.dev/injection-
```

File: modules/serverless-rn-1-10-0.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+[id="serverless-rn-1-10-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.10.0
+
+[id="new-features-1-10-0_{context}"]
+== New features
+* {ServerlessProductName} now uses Knative Operator 0.16.0.
+* {ServerlessProductName} now uses Knative Serving 0.16.0.
+* {ServerlessProductName} uses Knative Eventing 0.16.0.
+* {ServerlessProductName} now uses Kourier 0.16.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.16.1.
```
Comment (markusthoemmes) [inline modules/serverless-rn-1-10-0.adoc:14]: I'm in active discussions with upstream regarding their usage of patch versions and how much of a PITA it is to herd the zoo of releases.

Maybe we should opt to not specify down to the patch version here?

Comment (abrennan89) [reply modules/serverless-rn-1-10-0.adoc:14]: @markusthoemmes as long as this list is correct I'd include it as is for now, but for next release we can change our approach?

File: modules/serverless-deleting-broker-admin.adoc (hunk 2 above)
Comment (abrennan89) [reply modules/serverless-deleting-broker-admin.adoc:17]: Thanks @cardil

File: modules/serverless-rn-1-10-0.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* The annotation `knative-eventing-injection=enabled` that was previously used to annotate namespaces for broker creation is now deprecated. The new annotation is `eventing.knative.dev/injection=enable`. For more information, see the documentation on _Creating a broker as a cluster administrator using namespace annotation_.
```
Comment (mgencur) [inline modules/serverless-rn-1-10-0.adoc:15]: I'm curious if the "New Features" heading should be right above this line. Or the list of versions is considered as "new features"? ...If I look at the rendered document we are not consistent with this across releases.

Comment (abrennan89) [reply modules/serverless-rn-1-10-0.adoc:15]: So the list always goes under new features, it's supposed to be consistent for all releases but I see that it's not in 1.9.0, so let me fix that one.

**Comment by mgencur** [comment]
QE Approved. Thanks

**Review by sfortner-RH — APPROVED** [review]
LGTM!

**Pull request merged**